charts/be-web/Chart.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: v2
+type: application
+name: be-web
+description: Generic web BE application
+
+version: 0.1.0
+appVersion: latest
```

Comment:
As we cannot change this "on the fly" (with values.yaml), what about deleting this and use a custom field in `values.yaml`?